airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryStagingConsumerFactory.java

```diff
@@ -155,14 +144,12 @@ private OnStartFunction onStartFunction(final BigQueryStagingOperations bigQuery
                                           final TyperDeduper typerDeduper) {
     return () -> {
       LOGGER.info("Preparing airbyte_raw tables in destination started for {} streams", writeConfigs.size());
-      if (TypingAndDedupingFlag.isDestinationV2()) {
-        typerDeduper.prepareTables();
-      }
+      typerDeduper.prepareTables();
       for (final BigQueryWriteConfig writeConfig : writeConfigs.values()) {
         LOGGER.info("Preparing staging are in destination for schema: {}, stream: {}, target table: {}, stage: {}",
             writeConfig.tableSchema(), writeConfig.streamName(), writeConfig.targetTableId(), writeConfig.streamName());
         // In Destinations V2, we will always use the 'airbyte' schema/namespace for raw tables
-        final String rawDatasetId = TypingAndDedupingFlag.isDestinationV2() ? DEFAULT_AIRBYTE_INTERNAL_NAMESPACE : writeConfig.datasetId();
+        final String rawDatasetId = DEFAULT_AIRBYTE_INTERNAL_NAMESPACE;
```

Comment:
nit and paranoia:
nit: `// In Destinations V2, we will always use the 'airbyte_internal' schema/namespace for raw tables or an override if provided`
paranoia: We are using the override somewhere?

Reply:
... probably not. this seems like an actual bug.

Reply:
we ran some manual tests and this seems to work as expected. https://airbytehq-team.slack.com/archives/C03C4AVJWG4/p1693333526355829